execution_workflows/diffUTR/diffUTR_nextflow/nextflow.config

```diff
@@ -0,0 +1,131 @@
+/*
+ * -------------------------------------------------
+ *  APAeval Nextflow config file
+ * -------------------------------------------------
+ * Default config options for all environments.
+ */
+
+// Global default params, used in configs
+params {
+
+  // Options: Mandatory inputs (add more if required)
+  input                      = 'input.csv'
+
+
+  // Options: Custom config
+  custom_config_version      = 'master'
+  custom_config_base         = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
+  hostnames                  = [:]
+  config_profile_description = ''
+  config_profile_contact     = ''
+  config_profile_url         = ''
+
+  // Options: Other
+  help                       = false
+  outdir                     = './results'
+  enable_conda               = false
+  publish_dir_mode           = 'copy'
+  tracedir                   = "${params.outdir}/pipeline_info"
+  email                      = ''
+  email_on_fail              = ''
+  plaintext_email            = false
+  monochrome_logs            = false
+  name                       = ''
+  clusterOptions             = ''
+
+  // Defaults only, expecting to be overwritten
+  max_memory                 = '128.GB'
+  max_cpus                   = 16
+  max_time                   = '240.h'
+
+}
+
+// Container
+process.container = 'docker_container_name:tag'
+
+// Unmute the following if decided to create a docker container
+// Load base.config by default for all pipelines
+//includeConfig 'conf/base.config'
+
+profiles {
+  debug       { process.beforeScript = 'echo $HOSTNAME' }
+  docker      {
+    docker.enabled = true
+    // Avoid this error:
+    //   WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
+    // Testing this in nf-core after discussion here https://github.com/nf-core/tools/pull/351
+    // once this is established and works well, nextflow might implement this behavior as new default.
+    docker.runOptions = '-u \$(id -u):\$(id -g)'
+  }
+}
+
+// Unmute the following if decided to create a docker container
+// Export these variables to prevent local Python/R libraries from conflicting with those in the container
+env {
+  PYTHONNOUSERSITE = 1
+  R_PROFILE_USER = "/.Rprofile"
+  R_ENVIRON_USER = "/.Renviron"
+}
+
+// Capture exit codes from upstream processes when piping
+process.shell = ['/bin/bash', '-euo', 'pipefail']
+
+timeline {
+  enabled = true
+  file = "${params.tracedir}/execution_timeline.html"
+}
+report {
+  enabled = true
+  file = "${params.tracedir}/execution_report.html"
+}
+trace {
+  enabled = true
+  file = "${params.tracedir}/execution_trace.txt"
+}
+dag {
+  enabled = true
+  file = "${params.tracedir}/pipeline_dag.svg"
+}
+
+manifest {
+  name = 'APAeval'
```

Comment:
Replace *name*, *author*, *description* with the actual values. I guess @yuukiiwa can say what is expected here.